source/views/ViewStream.py

```diff
@@ -7,12 +7,23 @@
 from google.appengine.ext import blobstore
 from google.appengine.ext import ndb
 from google.appengine.ext.webapp import template
+from google.appengine.api import users
 
 stream_id_parm = 'streamID'
 
 
 class ViewStream(webapp2.RequestHandler):
     def get(self):
+        user = users.get_current_user()
```

Comment:
Thanks for putting this in. We should really get this very common piece of code into a global function somewhere!